lib/broccoli/ember-app.js

```diff
@@ -617,6 +619,41 @@ class EmberApp {
     }, []);
   }
 
+  _compileAddonTemplates(tree) {
+    if (registryHasPreprocessor(this.registry, 'template')) {
```

Comment:
Can we actually get here?

Reply:
removed

Reply:
I remember why I added it. Without it we get errors in the styles tests https://travis-ci.org/ember-cli/ember-cli/jobs/337422087#L805-L813 missing template preprocessor. I will add more mocking to the styles tests.

Reply:
added